gsms.py

```diff
@@ -53,15 +53,83 @@
 
 # standard imports
 import argparse
+import ctypes
+from ctypes import wintypes
 import json
 import os
+import re
 import shutil
 import time
+import uuid
 
 # lib imports
 import pylnk3
 
 
+# Code from here and modified to work in this project
+# https://gist.github.com/mkropat/7550097
+class GUID(ctypes.Structure):
+    """
+    Class to build a GUID compliant object for use in WIndows libraries
+
+    Parameters
+    -------
+    uuid: string
+
+    Returns
+    -------
+    GUID object
```

Comment:
class doesn't return, this is meant for functions with a `return` statement

Reply:
Will remove